src/index.html

```diff
@@ -28,27 +28,29 @@ <h3>Book a trip!</h3>
                 <div class = 'input'>
 
                   <div class='date-select'>
-                    <button class='user-entry-button' type="button" name="button"><img class = 'button-img' src='https://64.media.tumblr.com/92770b1f2bdb77c9dd213d2e8c272be3/1bc8f542ae83dc0f-f2/s540x810/b75b606ef073f8d993da1a9d913ee576a12312a3.png'></button>
+                    <!-- <button class='user-entry-button' type="button" name="button"><img class = 'button-img' src='https://64.media.tumblr.com/92770b1f2bdb77c9dd213d2e8c272be3/1bc8f542ae83dc0f-f2/s540x810/b75b606ef073f8d993da1a9d913ee576a12312a3.png'></button> -->
```

Comment:
If you don't need 'em, git rid of 'em! Don't let buttons hold you back!

Reply:
They have been n e r f e d.